setup.cfg

```diff
@@ -0,0 +1,14 @@
+[flake8]
+max-line-length = 90
+max-complexity = 10
+statistics = true
+application-import-names = user_deletion
+import-order-style = smarkets
+
+[coverage:run]
+source = user_deletion
+omit = *tests*
+
+[coverage:report]
+show_missing = True
+fail_under = 100
```

Comment:
`fail_under=100` fails on my local machine. It sounds like a bug, I'm going to remove it.